packages/devtools_app/test/treemap_test.dart

```diff
@@ -44,43 +44,128 @@ void main() {
 
     const windowSize = Size(2225.0, 1000.0);
 
-    testWidgetsWithWindowSize('builds treemap with expected data', windowSize,
-        (WidgetTester tester) async {
-      const treemapKey = Key('Treemap');
-      await pumpTreemapWidget(tester, treemapKey);
+    testWidgetsWithWindowSize(
+      'builds treemap with expected data',
+      windowSize,
+      (WidgetTester tester) async {
+        const treemapKey = Key('Treemap');
+        await pumpTreemapWidget(tester, treemapKey);
+
+        expect(find.byKey(treemapKey), findsOneWidget);
+
+        await expectLater(
+          find.byKey(treemapKey),
+          matchesGoldenFile('goldens/treemap_sizes.png'),
+        );
+        // Await delay for golden comparison.
+        await tester.pumpAndSettle(const Duration(seconds: 2));
+      },
+      skip: kIsWeb || !Platform.isMacOS,
+    );
+
+    testWidgetsWithWindowSize(
+      'builds treemap with expected data after zooming in',
+      windowSize,
+      (WidgetTester tester) async {
+        const treemapKey = Key('Treemap');
+        await pumpTreemapWidget(tester, treemapKey);
+
+        var text = 'package:flutter/src [1.82 MB]';
+        expect(find.text(text), findsOneWidget);
+        await tester.tap(find.text(text));
+        await tester.pumpAndSettle();
+
+        await pumpTreemapWidget(tester, treemapKey);
+
+        text = 'dart:ui [145 KB]';
+        expect(find.text(text), findsNothing);
+
+        await expectLater(
+          find.byKey(treemapKey),
+          matchesGoldenFile('goldens/treemap_sizes_zoom.png'),
+        );
+        // Await delay for golden comparison.
+        await tester.pumpAndSettle(const Duration(seconds: 2));
+      },
+      skip: kIsWeb || !Platform.isMacOS,
```

Comment:
you can add this skip to the whole group instead of individually skipping each test. Same for below.

Reply:
It seems like the tests aren't actually skipped even when I set the group skip parameters to a hard-coded `true`. I'm not sure if the result is intended; should I keep the skip parameters for each test?

Reply:
Hmm that is surprising. Go ahead and move the skips back to the tests.

Reply:
Done